composer.json

```diff
@@ -22,6 +23,7 @@
         "laravel/framework": "^8.12",
         "laravel/horizon": "^5.7",
         "laravel/nova": "^3.22",
+        "laravel/scout": "^8.6",
```

Comment:
I would encourage using [scout-extended](https://www.algolia.com/doc/framework-integration/laravel/getting-started/introduction-to-scout-extended/?client=php) in order to be able to customize the specificities of Algolia.

Reply:
Good point - will "switch". 👍

Reply:
https://github.com/Astrotomic/opendor.me/pull/72/commits/486ba90cac351620b2c9198df6c38829b91271be